pkg/store/kotsstore/app_store.go

```diff
@@ -15,6 +14,7 @@ import (
 	"github.com/replicatedhq/kots/pkg/logger"
 	"github.com/replicatedhq/kots/pkg/persistence"
 	troubleshootanalyze "github.com/replicatedhq/troubleshoot/pkg/analyze"
+	"github.com/rqlite/gorqlite"
```

Comment:
After the fact and not asking for changes, but you could alias `sql "github.com/rqlite/gorqlite"` to have fewer changes in the code :)

Reply:
ha, i'll keep that in mind for the next migration!